docs/source/quantization-support.rst

```diff
@@ -1,7 +1,7 @@
 Quantization API Reference
 -------------------------------
 
-torch.quantization
+torch.ao.quantization
```

Comment:
should we remove the `ao` for now, since the migration is not user facing yet?

Reply:
done